drivers/wdm/audio/backpln/portcls/dispatcher.cpp

```diff
@@ -6,13 +6,8 @@
  * PROGRAMMER:      Johannes Anderwald
  */
 
-#include "private.hpp"
-
-#ifndef YDEBUG
-#define NDEBUG
-#endif
 
-#include <debug.h>
+#include "private.hpp"
```

Comment:
strip one newline above #include "private.hpp" as it will help git compare to better align what should be aligned.

Reply:
Fixed.